aodndata/soop/soop_co2_classifier.py

```diff
@@ -0,0 +1,129 @@
+from aodncore.pipeline import FileClassifier
+from aodncore.pipeline.exceptions import InvalidFileNameError, InvalidFileContentError, InvalidFileFormatError
+from ship_callsign import ship_callsign_list, ship_callsign
+import datetime
+
+VALID_PROJECT = ['IMOS', 'FutureReefMap', 'SOOP-CO2_RT']
+VESSEL_CODE = {'AA': 'VNAA',
+               'IN': 'VLMJ'}
```

Comment:
Since all your code is conveniently packaged in a class, you could move these global variables into the class as well:
```python
class SoopCo2FileClassifier(FileClassifier):
    VALID_PROJECT = ['IMOS', 'FutureReefMap', 'SOOP-CO2_RT']
    VESSEL_CODE = {'AA': 'VNAA', 'IN': 'VLMJ'}
```
Then you can access them within the methods as e.g. `cls.VALID_PROJECT`